apps/frontend/pages/404.tsx

```diff
@@ -0,0 +1,33 @@
+import link from 'next/link'
+import styles from '../styles/404.module.css'
+
+export default function Custom404() {
+  return (
+    <div className={styles.wrapper}>
+
+      <h1 className={styles.mainH}><b>404.</b></h1>
```

Comment:
we don't use `h1, h2, h3, etc...` - check how to use `react-accesible-headings`

in this case, wrap an entire 404 in `<Level>` and use `<H>`